### PR TITLE
Close journal system properly when stopping Alluxio processes

### PR DESCRIPTION
**Pull request by ggezer**
This issue is encountered while authoring a new integration test for growing embedded journal cluster. Restarting an in-process master showed that journal system was not being closed during stopping standby masters.

### CODE REVIEW

**Comment by ggezer** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
@ggezer could you add some information in the description of the PR? Does this fix a bug, or is this for improving unittests?

File: minicluster/src/main/java/alluxio/master/LocalAlluxioJobCluster.java [hunk 1]
```diff
@@ -71,9 +74,12 @@ public void start() throws IOException, ConnectionFailedException {
    * @throws Exception when the operation fails
    */
   public void stop() throws Exception {
-    LOG.info("Stop Alluxio job service");
-    mWorker.stop();
-    mMaster.stop();
+    if (mIsRunning) {
```
Comment (gpang) [inline minicluster/src/main/java/alluxio/master/LocalAlluxioJobCluster.java]: what happens if you call stop when it is not running?

Comment (ggezer) [reply minicluster/src/main/java/alluxio/master/LocalAlluxioJobCluster.java]: Then it's a no-op as it should be. Or at least it's safe for it to be a no-op when not started.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**